src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
It is better not to use `*` selector(except box-sizing property). If you want to reset default paddings and margins, better to specify for which tags you want to reset them